### ISSUE TITLE
Error after 'Import From Evernote' KeyError: 'Evernote GUID'

### ISSUE THREAD

**Issue by djawesomesauce** [brumar/anknotes#21]
Hello,

I installed the add-on. I am running Windows 10 and using Anki v 2.0.44. When I click Tools > Import from Evernote the following error messages pops up:

An error occurred in an add-on. 
Please post on the add-on forum: 
https://anki.tenderapp.com/discussions/add-ons

Traceback (most recent call last): 
  File "C:\Users\Admin\AppData\Roaming\Anki2\addons\anknotes\__main__.py", line 334, in main 
    controller = Controller() 
  File "C:\Users\Admin\AppData\Roaming\Anki2\addons\anknotes\__main__.py", line 260, in __init__ 
    self.anki.add_evernote_model() 
  File "C:\Users\Admin\AppData\Roaming\Anki2\addons\anknotes\__main__.py", line 128, in add_evernote_model 
    source_ord, source_field = fmap[GUID_FIELD_NAME] 
KeyError: 'Evernote GUID'

**Comment by brumar** [comment]
Thank you for reporting this error. I ll look at it.

**Comment by brumar** [comment]
This is really weird. Did you have edited the evernote_note model ? Can you tell me what are the different fields of your "evernote_note" model (you can find it by adding manually a card and selecting type : evernote_note or in the browser mode (ctrl+B) in the models section.

**Comment by luxifer-art** [comment]
I did restart anki after the plugin installation. 

The fields for the evernote_note card type/model are "title," "content", and "Evernote Guid"
![cardscreenshot](https://user-images.githubusercontent.com/11995132/32858493-ab02deba-ca19-11e7-8a3c-963ceb42628d.png)

**Comment by brumar** [comment]
Thanks ! It should be "Evernote GUID" not "Evernote Guid". Can you click on "Fields" and try to rename it to see if it works ?
This error sounds mysterious. Do you remember renaming this field ? 
Can you create a new anki profile and try to use the addon again to see if you got the same problem ?

**Comment by luxifer-art** [comment]
Following your instructions to modify the field from "Evernote GUID" not "Evernote Guid" fixed the issue! Thank you. 

I don't remember renaming the field or even getting far enough to modify the evernote_note type.

I saw the same error on two different computers that had Anki desktop installed on them. 

I made a new profile after the fix and I didn't get the same problem. Thanks for the quick response with the problem.
